script.js

```diff
@@ -318,7 +318,13 @@ function addEventListeners(
  *
  */
 function addUpStudentCredits(reportCardTableElement) {
-  // code goes here
+  let creditElems = reportCardTableElement.querySelectorAll(".credit");
+  let creditSum = 0;
+  for (let credit of creditElems) {
```

Comment:
Using `for...of` is completely find since the return type of `querySelectorAll` is a NodeList.

Are you familiar with array functions like reduce, map, filter, etc.? Another option is to convert `creditElems` to an Array and reduce the `creditElems` list into `creditSum`. For example:

```
const creditElems= reportCardTableElement.querySelectorAll(".credit");
const creditSum = Array.from(creditElems).reduce((sum, credit) => sum + parseInt(credit.innerHTML), 0);
```

reduce - https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/reduce